### PR TITLE
Add basic elixir debugging tips

### PR DESCRIPTION
**Pull request by rtorrero**
This PR adds some basic tips on how to use `pry` to debug in elixir.

### CODE REVIEW

**Comment by rtorrero** [comment]
I had this added a while ago and realized I never put anyone assigned for review :sweat_smile: is it still something you think might be useful or should we drop it?

File: coding-standards/elixir.md [hunk 1]
```diff
@@ -99,3 +99,72 @@ Guidelines for applications using Phoenix:
...
+```
+  def handle(%DeregistrationProcessManager{}, %HostDeregistrationRequested{
+        host_id: host_id,
+        requested_at: requested_at
+      }) do
+    %DeregisterHost{host_id: host_id, deregistered_at: requested_at}
+  end
+```
+
+we would add the breakpoint as explained above:
```
Comment (EMaksy) [inline coding-standards/elixir.md:128]: Maybe a bit more direct ? 
Add  `require IEx` and `IEx.pry()`  as a breakpoint.
Example:

What do you think?

File: coding-standards/elixir.md [hunk 2]
```diff
@@ -99,3 +99,72 @@ Guidelines for applications using Phoenix:
...
+        host_id: host_id,
+        requested_at: requested_at
+      }) do
+    require IEx
+    IEx.pry()
+    %DeregisterHost{host_id: host_id, deregistered_at: requested_at}
+  end
+```
+
+once we trigger any action that causes the breakpoint to be reached, you should see something like:
```
Comment (EMaksy) [inline coding-standards/elixir.md:141]: Start with Capital letter here 

`once --> Once`

Also "you should see" is a bit unprecise or what do you think ?

**Review by EMaksy — COMMENTED** [review]
Hey man, good job, I left some suggestions :) 
You inspired me to play with the debugger, thanks!

**Pull request closed without merging**